Cargo.toml

```diff
@@ -103,5 +103,12 @@ serde_json = "1.0.96"
 triomphe = { version = "0.1.8", default-features = false, features = ["std"] }
 # can't upgrade due to dashmap depending on 0.12.3 currently
 hashbrown = { version = "0.12.3", features = [
-    "inline-more",
+  "inline-more",
 ], default-features = false }
+
+rustc_lexer = { version = "0.10.0", package = "ra-ap-rustc_lexer" }
+rustc_parse_format = { version = "0.10.0", package = "ra-ap-rustc_parse_format", default-features = false }
+
+# Upstream broke this for us so we can't update it
+rustc_abi = { version = "0.0.20221221", package = "hkalbasi-rustc-ap-rustc_abi", default-features = false }
+rustc_index = { version = "0.0.20221221", package = "hkalbasi-rustc-ap-rustc_index", default-features = false }
```

Comment:
I think you messed up your rebase?

Reply:
Oops thank you for pointing it out. It was removed in this [PR](https://github.com/rust-lang/rust-analyzer/commit/f4704bc8ae92d150b2801e391692d9503f0c6126#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542L104-R107) so removed in [1e11a55](https://github.com/rust-lang/rust-analyzer/pull/15618/commits/1e11a55f98d9b3f1f7ac2a6779e10b060fe3c896).